src/domain/landingPage/landingPage.module.scss

```diff
@@ -8,5 +12,53 @@
 }
 
 .searchContainer {
-  background-color: var(--color-white);
+  // act as a container when desktop size
+  @extend .container;
+  margin-top: -3rem;
+
+  @include respond-above(md) {
+    background: linear-gradient(
+      90deg,
+      var(--color-copper-medium-light) 50%,
+      var(--color-fog-medium-light) 50%
+    );
+    background-color: var(--color-white);
+    clip-path: url(#wave);
```

Comment:
needed to check the browser compability and it looks good: https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path